src/Scripting/CSharpTest.Desktop/CsiTests.cs

```diff
@@ -47,7 +48,17 @@ public void CurrentWorkingDirectory_Change()
 new C()
 Environment.Exit(0)
 ";
-            var result = ProcessUtilities.Run(CsiPath, "", stdInput: stdInput);
+            ProcessResult result = null;
```

Comment:
I agree with @tmat that a general fix would be preferable to a specific fix.  However, before creating a new encoding, you might want to have a look for other references to that constructor - we've probably wanted no-BOM UTF8 before.